backend/src/test/java/gov/cdc/usds/simplereport/service/ApiUserServiceTest.java

```diff
@@ -208,6 +214,30 @@ void createUserInCurrentOrg_disabledUserWrongOrg_error() {
     assertEquals("Unable to add user.", caught.getMessage());
   }
 
+  @Test
+  @WithSimpleReportSiteAdminUser
+  void getAllActiveUsersByOrganization_success() {
+    Organization org = _dataFactory.createValidOrg();
+    _dataFactory.createValidApiUser("allfacilities@example.com", org);
+    _dataFactory.createValidApiUser("nofacilities@example.com", org);
+    UserInfo userToBeDeleted = _dataFactory.createValidApiUser("somefacilities@example.com", org);
+    // delete a user
```

Comment:
nit: I don't think you need the comments here, just space appropriately to indicate different operations 😸

Reply:
Also, I might assert before and after delete to make the delete step more meaningful - or else the delete could be broken out into a separate test.

Reply:
Ok! Will clean up the comments 🧹  and will add some assertions! Thank you, George! 😹

Reply:
OOO this test is actually wrong!! oops my bad. It is suppose to return the deleted user and the  active users... but it looks like since I didn't update the `DemoOktaRepository.java` file it is not doing what I expected... I'll rename the test and fix this up! 😅